contracts/payloads/IGP83/PayloadIGP83.sol

```diff
@@ -0,0 +1,666 @@
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_ezETH_ETH); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(ezETH_ETH_DEX, TEAM_MULTISIG, false);
+            }
+        }
+
+        {
+            address ezETH__wstETH_VAULT = getVaultAddress(103);
+
+            // [TYPE 1] ezETH<>wstETH | normal collateral & normal debt
+            Vault memory VAULT_ezETH_wstETH = Vault({
+                vault: ezETH__wstETH_VAULT,
+                vaultType: TYPE.TYPE_1,
+                supplyToken: ezETH_ADDRESS,
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000_000, // $10M
+                baseBorrowLimitInUSD: 10_000_000, // $10M
+                maxBorrowLimitInUSD: 30_000_000 // $30M
+            });
+
+            setVaultLimits(VAULT_ezETH_wstETH); // TYPE_1 => 103
+
+            VAULT_FACTORY.setVaultAuth(
+                ezETH__wstETH_VAULT,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+
+        {
+            address ezETH_ETH__wstETH_VAULT = getVaultAddress(104);
+
+            // [TYPE 2] ezETH-ETH<>wstETH | smart collateral & normal debt
+            Vault memory VAULT_ezETH_ETH_wstETH = Vault({
+                vault: ezETH_ETH__wstETH_VAULT,
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 0,
+                baseBorrowLimitInUSD: 7_500_000, // $7.5M
+                maxBorrowLimitInUSD: 15_000_000 // $15M
+            });
+
+            setVaultLimits(VAULT_ezETH_ETH_wstETH); // TYPE_2 => 104
+
+            VAULT_FACTORY.setVaultAuth(
+                ezETH_ETH__wstETH_VAULT,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+    }
+
+    // @notice Action 5: Update cbBTC-wBTC DEX configs
+    function action5() internal isActionSkippable(5) {
+
+        address cbBTC_wBTC_DEX_ADDRESS = getDexAddress(3);
+
+        // update the threshold to 20%
+        IFluidDex(cbBTC_wBTC_DEX_ADDRESS).updateThresholdPercent(
+            20 * 1e4,
+            20 * 1e4,
+            9 hours,
+            1 days
+        );
+
+        // update the upper and lower range +-0.25%
+        IFluidDex(cbBTC_wBTC_DEX_ADDRESS).updateRangePercents(
+            0.25 * 1e4,
+            0.25 * 1e4,
+            1 days
+        );
+    }
+
+    // @notice Action 6: Update wstETH-ETH, weETH-ETH, rsETH-ETH DEX configs
+    function action6() internal isActionSkippable(6) {
+
+        address wstETH_ETH_DEX_ADDRESS = getDexAddress(1);
+        address weETH_ETH_DEX_ADDRESS = getDexAddress(9);
+        address rsETH_ETH_DEX_ADDRESS = getDexAddress(13);
+
+        // update the upper and lower range for wstETH-ETH DEX
+        IFluidDex(wstETH_ETH_DEX_ADDRESS).updateRangePercents(
+            0.08 * 1e4, // +0.08%
```

Comment:
all here should switch, 

upper is always `1` 

lower should be the config value

Note 

 ```
 
    /// @param upperPercent_ in 4 decimals, 10000 = 1%
    /// @param lowerPercent_ in 4 decimals, 10000 = 1%
    /// @param shiftTime_ in secs, in how much time the upper percent configs change should be fully done
    function updateRangePercents(
        uint upperPercent_,
        uint lowerPercent_,
        uint shiftTime_
    ) public _check _onlyDelegateCall {
     ```